src/style.css

```diff
@@ -1 +1,91 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header,
+.nav_list {
+  display: flex;
+  align-items: center;
+}
+
+.header {
+  font-family: inherit;
+  font-weight: 500;
+  font-style: normal;
+  justify-content: space-between;
+  position: relative;
+  height: 60px;
+  width: 100%;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05) ;
+}
+
+.header__link {
+  display: flex;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 50px;
+}
+
+.header__nav {
+  margin-right: 50px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav_list-item {
+  display: inline-block;
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: inline-block;
+  padding: 23px 0;
+  font-family: inherit;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  text-decoration: underline;
+  text-decoration-color: #00acdc;
+}
+
+.is-active {
+  display: inline-block;
```

Comment:
Remove the unnecessary `display: inline-block;` and `position: relative;` properties since they are already defined for the `.nav__link` class.